src/js/dnt-helper.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+var dntEnabled = require("@mozmeao/dnt-helper");
```

Comment:
Any reason why this needs to be in a separate script? Could you just import it directly into `gtm-ga.js`, and avoid the need for the global variables?